script/ghci-flags-dependencies

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Computes the paths to files causing changes to the ghci flags. You probably won’t be running this yourself, but rather ghcide will via configuration in hie.yaml.
```

Comment:
Changes to the listed files might result in different package flags for `ghcide` to learn of, so we tell it to watch them.